tests/audit/io_test.py

```diff
@@ -12,7 +12,7 @@
                 'allow_backstep': True,
             },
             (
-                'Is this a valid secret (not a false-positive)? '
+                'Is this a secret that should be committed to this repository? '
```

Comment:
This seems much clearer to me although let me just give an example to make sure I understand it correctly.

If I had accidentally copied a production AWS key into my code that we wouldn't want to fall into the wrong hands and the scan found it and asked the question 'Is this a secret that should be committed to this repository?', then I would answer 'no'. Have I understood that correctly?

One thing I wanted to check, if I had been asked 'Is this a valid secret (not a false-positive)?' instead I would have answered 'yes'. Does this mean that the 'yes' and 'no' options have now swapped?

Reply:
🤦 You're absolutely right! This language would swap the `yes` and `no` options. Let me fix this.

Reply:
Thanks, so just to make sure I understand, we are going to go with this new wording (which I'm happy with) but it will require you to go and make an additional change to swap the logic for `yes` and `no` options?

Reply:
Ah infact, I see you added that fix as a commit in this PR. Great, this is looking good!